lib/parse.js

```diff
@@ -17,21 +18,21 @@ module.exports = parse;
  * The promise resolves with the parsed file contents, NOT the raw (Buffer) contents.
  */
 async function parse (path, $refs, options) {
-  try {
```

Comment:
Any reason not to put the logic below (stripHash, getExtension, etc) into the try?

Reply:
It's an async function anyway and we throw in `catch` (and the old code also threw), so the behavior is the same.